modules/navier_stokes/test/tests/finite_volume/pins/physics/heated_porous_medium.i

```diff
@@ -0,0 +1,150 @@
+# Fluid properties
+mu = 1.1
+rho = 1.1
+h_fs = 100
+k = 0.9
+cp = 3000
+
+# Solid properties
+rho_solid = 1.1
+k_solid = 13
+cp_solid = 1000
+
+[Problem]
+  error_on_jacobian_nonzero_reallocation = true
+[]
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 5
+    ymin = 0
+    ymax = 1
+    nx = 20
+    ny = 10
+  []
+[]
+
+[Modules]
+  [NavierStokesFV]
```

Comment:
Do you want to update this to use your physics?

Reply:
done